src/_posts/languages/nodejs/2000-01-01-start.md

```diff
@@ -92,19 +92,11 @@ Buildpack]({% post_url platform/deployment/buildpacks/2000-01-01-ssh-key %}),
 this buildpack will let you setup a private SSH key securely in the build
 environment.
 
-### Install devDependencies
+### devDependencies Installation
 
-By default, dependencies present in the `devDependencies` field are not
-installed since they should not be needed for production. If you want to
-install them, you'll need to set the `NPM_CONFIG_PRODUCTION` environment
-variable to false via our web interface or by using our CLI:
+By default, dependencies present in the `devDependencies` field are installed. At the end of the deployment, `devDependencies` are pruned. Hence only production dependencies are left in the image used for the runtime.
 
-```bash
-$ scalingo --app my-app env-set NPM_CONFIG_PRODUCTION=false
-```
-
-After setting the NPM_CONFIG_PRODUCTION environment variable, you have to
-redeploy your code.
+The pruning step can be skipped by setting the environment variable `YARN_PRODUCTION` to any value.
```

Comment:
Sure? Setting `YARN_PRODUCTION` to `true` shouldn't skip

Reply:
I'm pretty sure, yes:

https://github.com/Scalingo/nodejs-buildpack/blob/5169a57250df80a532f1aa9c4e19b8bb18ed3b46/lib/dependencies.sh#L176-L179

Reply:
Seems a bit counter-intuitive, but you're right

Reply:
Thanks for paying attention to this detail :)